cmd/livepeer/livepeer.go

```diff
@@ -820,6 +834,20 @@ func validateURL(u string) (*url.URL, error) {
 	return p, nil
 }
 
+func isLocalURL(u string) (bool, error) {
+	uri, err := url.ParseRequestURI(u)
+	if err != nil {
+		return false, err
+	}
+
+	hostname := uri.Hostname()
+	if hostname == "127.0.0.1" || hostname == "localhost" {
```

Comment:
Technically, the entire 127.0.0.0/8 range [1] (or 127.0.0.1/32, depending on who you're talking to [2])  is reserved for loopback. In practice, it probably doesn't really matter because our default is `127.0.0.1`.

[1] RFC 6890, Section 2.2.2. Table 4, "Loopback" https://tools.ietf.org/html/rfc6890

[2] RFC 3330, Section 2  "Global and Other Specialized Address Blocks" https://tools.ietf.org/html/rfc3330

Reply:
Fixed in c299a2a